lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -76,55 +76,55 @@
         <h3 class="course-title">
           % if show_courseware_link:
             % if not is_course_blocked:
-              <a href="${course_target}">${course.display_name_with_default}</a>
+              <a href="${course_target}">${course_overview.display_name_with_default}</a>
             % else:
-              <a class="disable-look">${course.display_name_with_default}</a>
+              <a class="disable-look">${course_overview.display_name_with_default}</a>
             % endif
           % else:
-            <span>${course.display_name_with_default}</span>
+            <span>${course_overview.display_name_with_default}</span>
           % endif
         </h3>
         <div class="course-info">
-          <span class="info-university">${get_course_about_section(course, 'university')} - </span>
```

Comment:
The original reason I made that change is because `get_course_about_section(course, 'university')` is the same thing as `course.display_org_with_default`, and I didn't want to have to make `get_course_about_section` support CourseOverviews. In hindsight this is a bad idea, because the implementation of `get_course_about_section` might change. 

My new solution is making a common function `get_course_university_about_section`.